toolset/run-tests.py

```diff
@@ -189,6 +189,17 @@ def main(argv=None):
         default=None,
         help='The network mode to run docker in')
 
+    parser.add_argument(
+        '--local-dev-mode',
+        action='store_true',
+        default=False,
+        help='server and database both run locally, very easy during development, building containers takes huge aount of time every single time if there are changes in framework source code')
```

Comment:
We don't want to support `--local-dev-mode`, but there is a decent alternative for running the server locally.  You can pass in the `--server-host` argument with a hostname or IP address that resolves to your workstation, just something other than localhost/127.0.0.1.  If you do that and you temporarily replace your dockerfile with something like this:
```
FROM buildpack-deps:xenial
CMD sleep infinity
```
then you can skip the server build times and hit your local server instance.

I don't think the same approach will work for the database, but you shouldn't have to build any of the databases more than once.  I think the changes in this PR to the databases was maybe coming from confusion about the cached query test requirements - it's supposed to be backed by the same worlds table as the other tests.